include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -664,395 +659,197 @@ namespace picongpu
                 }
 
 
-                /** Write Amplitude data to HDF5 file
+                /** Write Amplitude data to openPMD file
                  *
                  * Arguments:
                  * Amplitude* values - array of complex amplitude values
                  * std::string name - path and beginning of file name to store data to
                  */
-                void writeHDF5file(std::vector<Amplitude>& values, std::string name)
+                void writeOpenPMDfile(std::vector<Amplitude>& values, std::string name)
                 {
-                    splash::SerialDataCollector hdf5DataFile(1);
-                    splash::DataCollector::FileCreationAttr fAttr;
-
-                    splash::DataCollector::initFileCreationAttr(fAttr);
-                    fAttr.enableCompression = compressionOn;
-
                     std::ostringstream filename;
-                    filename << name << currentStep;
-
-                    hdf5DataFile.open(filename.str().c_str(), fAttr);
-
-                    typename PICToSplash<Amplitude::complex_T::type>::type radSplashType;
-
-
-                    splash::Dimensions bufferSize(
-                        Amplitude::numComponents,
-                        radiation_frequencies::N_omega,
-                        parameters::N_observer);
-
-                    splash::Dimensions componentSize(1, radiation_frequencies::N_omega, parameters::N_observer);
-
-                    splash::Dimensions stride(Amplitude::numComponents, 1, 1);
+                    // TODO: needs to be changed to ".h5" and also support adios
+                    filename << name << "%T_0_0_0.h5";
```

Comment:
I am a bit confused, perhaps also by earlier HDF5 requirement documentation. Would it be enough to rename file to `.bp`? Or does the code relies on HDF5 somewhere else?

Reply:
Yes, the python module to read the amplitude data is currently based on h5py and not on openPMD-api.

Reply:
But since the python module does not rely on the serial libSplashending `_0_0_0.h5`, I will remove it now.

Reply:
Just to be sure, you mean to keep the `%T` part?
I also always dropped the `_0_0_0` things, but the `%T` is what makes it create a new file per iteration.

Reply:
If hdf5 is only a requirement of the postprocessing, perhaps it should not be a hard error here, and a note in documentation is sufficient?

However I don't insist, you know better how it's realistically used.

Reply:
Also addressing post-processing
Does that use libsplash? If yes, we should consider that as part of the efforts to get libsplash removed
For some plugin (I don't remember which one), I converted the postprocessing to openPMD, making both HDF5 and ADIOS viable backends for writing and reading
Anyhow, did you check if the postprocessing can still read the files that you are writing?

Reply:
I agree with your general point @franzpoeschel . I think in this case the postprocessing is just h5py.